clients/fides-js/src/components/fides.css

```diff
@@ -1214,12 +1227,6 @@ div#fides-overlay-wrapper .fides-i18n-pseudo-button {
   max-height: 7rem;
   transition: height 0.5s;
 }
-@media screen and (max-width: 768px) {
-  .fides-i18n-popover {
-    bottom: auto;
-    top: 100%;
-  }
-}
```

Comment:
This was causing the language switcher menu to open down instead of up. Must have been leftover from a time when the switcher was at the top of the button stack instead of the bottom